src/Compilers/CSharp/Test/Emit3/FieldKeywordTests.cs

```diff
@@ -1053,6 +1053,7 @@ static void Main()
                 }
                 """;
             var verifier = CompileAndVerify(source, expectedOutput: "(1, 2, 3, 5, 0, 9, 11)");
+            verifier.Compilation.VerifyDiagnostics();
```

Comment:
verifier itself has a VerifyDiagnostics method that will do a little less work, I think, and also include any emit diagnostics.